metadata/src/main/java/org/apache/kafka/metadata/migration/KRaftMigrationDriver.java

```diff
@@ -635,12 +635,24 @@ class BecomeZkControllerEvent extends MigrationEvent {
         public void run() throws Exception {
             if (checkDriverState(MigrationDriverState.BECOME_CONTROLLER, this)) {
                 applyMigrationOperation("Claiming ZK controller leadership", zkMigrationClient::claimControllerLeadership);
-                if (migrationLeadershipState.zkControllerEpochZkVersion() == -1) {
+                if (migrationLeadershipState.zkControllerEpochZkVersion() == -2) {
```

Comment:
Can we have a constant for this like `ZkMigrationLeadershipState.UNKNOWN_ZK_VERSION`?

Reply:
There's already sort a constant in ZkMigrationLeadershipState.EMPTY